src/test/kotlin/com/exactpro/th2/check1/rule/check/TestCheckRuleTask.kt

```diff
@@ -307,6 +314,85 @@ internal class TestCheckRuleTask : AbstractCheckTaskTest() {
         })
     }
 
+    @ParameterizedTest
+    @ValueSource(ints = [0, 1, 2, 3, 4])
+    fun `simple collection order checking config`(testCase: Int) {
```

Comment:
Instead of this you can use `@MethodSource` annotation and provide the required arguments in a method instead of using an array inside the method